src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -79,20 +86,17 @@ private static void setupLogger(Application application) {
 	 * Launch the application.
 	 */
 	public static void main(String[] args) {
-		EventQueue.invokeLater(new Runnable() {
-			public void run() {
-				Application app = new Application("2d jobs Visio");
-				DrawerFeature.setupDrawerPlugin(app);
-				setupDefaultDrawerVisibilityManagement(app);
+		EventQueue.invokeLater(() -> {
+            Application app = new Application("2d jobs Visio");
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.